user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Extensions/API_Reference/IGQILogger.md

```diff
@@ -0,0 +1,79 @@
+---
+uid: GQI_IGQILogger
+---
+
+# IGQILogger interface
+
+## Definition
+
+- Namespace: `Skyline.DataMiner.Analytics.GenericInterface`
+- Assembly: `SLAnalyticsTypes.dll`
+
+Contains functionality to log messages and exceptions. The log files will be placed within the `C:\Skyline DataMiner\Logging\GQI\Extensions` folder.
+
+## Properties
+
+| Property | Type | Description |
+| -------- | ---- | ----------- |
+| MinimumLogLevel | GQILogLevel | Set or get the current minimum log level. The default value is `Information`. |
```

Comment:
We should make an API reference for the GQILogLevel as well, and link to it here.